### PR TITLE
Slackbot and retweetbot.

### PR DESCRIPTION
**Pull request by RML-Admin**
Added slackbot for approval. Retweetbot can be refactored and hookedup for actual retweet.

### CODE REVIEW

File: twote/slackbot.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                   '#thankful,#gratitude,#healthy,#yoga,#positivity,#community,#ecosystem,#planet,#meditation,#bliss,'
+                   ).split(',')
+
+
+_message = {
+    "text": "This is the tweet!",
+    "attachments": [
+        {
+            "text": "Retweet?",
+            "fallback": "You are unable to choose a game",
```
Comment (hobson) [inline twote/slackbot.py:32]: CRUFT, but not a problem

File: twote/slackbot.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                   '#social,#economics,#prosocial,#peaceandcookies,#humility,#shoutout,' +
+                   '#opendata,#openscience,#openai,#opensource,' +
+                   '#data,#dataviz,#d3js,#datascience,#machinelearning,#ai,#neuralnet,#deeplearning,#iot,' +
+                   '#hack,#hacking,#hackathon,#compsci,#coding,#coder,' +
+                   '#mondaymotivation,#motivation,#life,#mind,' +
+                   '#play,#game,#logic,#gametheory,#winning,' +
+                   '#kind,#bekind,#hope,#nice,#polite,#peace,#inspired,#motivated,#inspiration,#inspiring,#quote,' +
+                   '#awesome,#beawesome,#payitforward,#give,#giving,#giveandtake,#love,#pause,#quiet,' +
+                   '#windows,#linux,#ubuntu,#osx,#android,#ios,' +
+                   '#thankful,#gratitude,#healthy,#yoga,#positivity,#community,#ecosystem,#planet,#meditation,#bliss,'
```
Comment (hobson) [inline twote/slackbot.py:23]: Nice list of topics. Thanks for taking out all the negative ones :)

File: twote/slackbot.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+					"style" : "danger",
+                    "type": "button",
+                    "value": "reject"
+                },
+                {
+                    "name": "another",
+                    "text": "Send another one",
+
+                    "type": "button",
+                    "value": "war",
```
Comment (hobson) [inline twote/slackbot.py:62]: CRUFT, no worries... yet.

File: twote/slackbot.py (hunk 1 above)
Comment (RML-Admin) [reply twote/slackbot.py:32]: oh yeah! I like to get something working and then cleaning it up :). Suggest changes.

**Pull request merged**